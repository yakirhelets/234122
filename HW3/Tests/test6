getApartments sourceOne Eminem 4
