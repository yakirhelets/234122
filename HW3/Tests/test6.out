12 3000 050-1111110
123 3000 050-1111110
12 3000 050-1111110
11 2000 050-1111110
